internal/wrappers/wrappers.go

```diff
@@ -52,6 +54,16 @@ type ISCSILib interface {
 	CreateOrUpdateNode(target goiscsi.ISCSITarget, options map[string]string) error
 }
 
+type NVMeTCP interface {
+	DiscoverNVMeTCPTargets(address string, login bool) ([]gonvme.NVMeTarget, error)
+	GetInitiators(filename string) ([]string, error)
+	NVMeConnect(target gonvme.NVMeTarget) error
+	NVMeDisonnect(target gonvme.NVMeTarget) error
```

Comment:
Please correct the typo in nvme disconnect. It has also been corrected in the gonvme library now.

Reply:
changed to Disconnect